textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/team3lucenequeryexample/LuceneQueryExample.java

```diff
@@ -0,0 +1,78 @@
+package edu.uci.ics.textdb.sandbox.team3lucenequeryexample;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.Tokenizer;
+import org.apache.lucene.analysis.ngram.NGramTokenizer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.FieldType;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexOptions;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+
+public class LuceneQueryExample {
+	public static void main(String[] args) throws Exception {
+		String dataFileName = "/Users/laishuying/Desktop/lucenetext.txt";
+		
+		Analyzer analyzer = new Analyzer() {
+			@Override
+			protected TokenStreamComponents createComponents(String fieldName) {
+				Tokenizer source = new NGramTokenizer(3,3);
+				return new TokenStreamComponents(source);
+			}
+		};
+		
+		Directory indexDir = FSDirectory.open(Paths.get("index"));
+		try {
+			IndexWriterConfig config = new IndexWriterConfig(analyzer);
+			IndexWriter writer = new IndexWriter(indexDir, config);
+			Scanner dataFile = new Scanner(new File(dataFileName));
+			Integer lineNum = 1;
+			while (dataFile.hasNextLine()) {
```

Comment:
I tried to use IDataReader/IDataWriter to populate a data store with a data file, but I can't get any results when searching my query.
The reason is that we need to set storeTermVectors = true when we create(write) data fields. However, in  [Utils.java](https://github.com/TextDB/textdb/blob/master/textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java), getLuceneField leaves storeTermVectors = false (default value).
In [Lucene FieldType Doc](https://lucene.apache.org/core/5_4_1/core/org/apache/lucene/document/FieldType.html), storeTermVectors = false means field's indexed form will NOT be stored into term vectors. In my understanding, this is causing IndexReader.getTermVector to return null and no search result is returned.
I got expected search results if using IndexWriter directly. I realise this will be a big thing if we really need to change IDataWriter. @zuozhi Could you also look into this?